src/routes/llms-full.txt/+server.ts

```diff
@@ -0,0 +1,95 @@
+import type { RequestHandler } from '@sveltejs/kit';
+
+export const prerender = true;
+
+const markdocAndMarkdownFiles = import.meta.glob('$routes/**/*.{markdoc,md}', {
+    query: '?raw',
+    import: 'default',
+    eager: true
+});
+
+function stripRouteGroups(routePath: string): string {
+    return routePath.replace(/\([^/]+\)\//g, '');
+}
+
+function pathToRoute(path: string): string {
+    let route = path.replace(/^\/src\/routes/, '');
+
+    if (/\/+page\.(markdoc|md)$/.test(route)) {
+        route = route.replace(/\/+page\.(markdoc|md)$/, '');
+    } else {
+        route = route.replace(/\.(markdoc|md)$/, '');
+    }
+
+    route = stripRouteGroups(route.replace(/\\/g, '/'));
+    return route || '/';
+}
+
+function stripFrontmatter(text: string): string {
+    return text.replace(/^\s*---\s*\n[\s\S]*?\n---\s*/m, '');
+}
+
+function sectionWeight(href: string): number {
+    try {
+        const u = new URL(href);
+        const path = u.pathname;
+        if (path.startsWith('/docs')) return 0;
+        if (path.startsWith('/blog')) return 1;
+        if (path.startsWith('/integrations')) return 2;
+        return 3;
+    } catch {
+        if (href.startsWith('/docs')) return 0;
+        if (href.startsWith('/blog')) return 1;
+        if (href.startsWith('/integrations')) return 2;
+        return 3;
+    }
+}
+
+export const GET: RequestHandler = ({ request }) => {
+    try {
+        type Item = { href: string; title: string; block: string };
+        const items: Item[] = [];
+        for (const path of Object.keys(markdocAndMarkdownFiles)) {
+            const raw = markdocAndMarkdownFiles[path] as string;
+
+            const route = pathToRoute(path);
+            if (route.includes('[')) continue;
+
+            const href = route.startsWith('/') ? route : `/${route}`;
+            const url = new URL(href, request.url).toString();
+
+            const titleMatch = raw.match(/^#\s+(.+)$/m);
+            const title = titleMatch
+                ? titleMatch[1].trim()
+                : href.split('/').pop()!.replace(/[-_]/g, ' ');
+
+            const body = stripFrontmatter(raw).trim();
+            items.push({ href: url, title, block: `## ${title}\n${url}\n\n${body}\n\n---` });
```

Comment:
title is wrong: 
<img width="704" height="468" alt="image" src="https://github.com/user-attachments/assets/26f74ac0-81ff-4326-9fc4-3de3236d4efc" />

Reply:
Also, we need to fix the heading levels. This doesn't match the [docs page](https://appwrite.io/docs/advanced/migrations) this is for.  

<img width="771" height="505" alt="image" src="https://github.com/user-attachments/assets/a88ab0b1-33de-4196-a9cd-36677676cf88" />